Tests/NIOTests/ChannelTests.swift

```diff
@@ -169,9 +169,15 @@ public class ChannelTests: XCTestCase {
         for _ in 0..<bufferSize {
             buffer.write(staticString: "a")
         }
+        
 
+        #if arch(arm) // 32-bit, Raspi/AppleWatch/etc
+            let lotsOfData = Int(Int32.max / 8)
```

Comment:
why do we use less data here? Is that documented to be the maximum?

Reply:
Why do you use `Int32.max` instead of `Int64.max` on 64-bit? Same reason ;-)

Reply:
but we're only allocating 2MB here, right? We'll write the same buffer over and over again, this should work. The sizes were chosen so it will exhaust both the number of bytes limit (IIRC 2GB) as well as the number of chunks (1024) you're allowed to give to `writev`. If we think this isn't worthwhile, then we can lower the number for all platforms. CC @Lukasa

Reply:
@weissi is correct, the purpose of this is to validate that we correctly handle having more data to pass to `writev` than `writev` will accept. As @weissi notes, we only actually allocate one buffer, so there is no risk.

Reply:
Oh, I see. You just repeatedly write the 2MB buffer. Maybe the overflow happens because of `<=`? I guess I need to check this one on a Raspi again.

Reply:
it'd be awesome if you could validate why it fails. There must be something else wrong.

Reply:
Well, see my () comment:
> internally it still has to spool up Int32.max of buffers, which won't work out